docs_list.md

```diff
@@ -0,0 +1,143 @@
+# Docs list
+
+A working list of docs with links to the published documentation, the repo or repos, and the intended purpose of the technical content.
+
+## Cosmos Hub
+
+- [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html)
+
+- <https://github.com/cosmos/gaia/tree/master/docs>
+
+- Using the Cosmos SDK application for the Cosmos Hub.
+
+  <details><summary>Top-level contents</summary>
+
+  - Cosmos Hub
+
+  - Delegators
+
+  - Gaia Tutorials
+
+  - Validators
+
+  </details>
+
+## Cosmos SDK Documentation
+
+- [Cosmos SDK Documentation](https://docs.cosmos.network/) _why 2 repos?_
+
+- <https://github.com/allinbits/cosmos-sdk/tree/master/docs>
+
+- <https://github.com/cosmos/cosmos-sdk/tree/master/docs> <!-- two repos, what is the difference? -->
```

Comment:
```suggestion
```

one is a fork by all in bits, not sure why it's there probably shoudl consider removing it since everyone has access to the cosmos org version

Reply:
@okwme this repo will be deleted, per Alessio and Sahith, so I am removing from this list